package.json

```diff
@@ -36,7 +36,7 @@
     "@babel/core": "^7.19.3",
     "@babel/preset-react": "^7.18.6",
     "@monaco-editor/react": "^4.4.6",
-    "monaco-editor-wrapper": "^1.2.0",
+    "monaco-editor-wrapper": "^1.3.1-next.0",
```

Comment:
We should switch to final 1.3.1 before merging. Also, if possible the whole `langium-preview-develop` branch should be rebased to the latest `main`.

Reply:
Since this is still preliminary development, I think going with next is okay for *now*, and then we can include the later **1.3.1** (so long as there's no major breaking changes). Unless that's being released in the next day or two, then we can incorporate it before merging here

However It would be best to get something added first, and then update, as it's currently blocking further work.

Reply:
1.3.1 is now available, so this won't block you if that is ok? 🙂